config/config.go

```diff
@@ -247,6 +247,7 @@ type Config struct {
 	ControlAPIPort                    int                                   `json:"control_api_port"`
 	EnableCustomDomains               bool                                  `json:"enable_custom_domains"`
 	EnableJSVM                        bool                                  `json:"enable_jsvm"`
+	JSVMTimeout						  int									`json:"jsvm_timeout"`
```

Comment:
how about json-tag for this field? we need some name declared so it will be easier to document. also, formatting - make sure you use `go fmt`

Reply:
sorry, just realized that I had to scroll a lot to the right to see `json:"jsvm_timeout"` :) is it formatting issue?

Reply:
@LuanvP you should apply gofmt to fix formatting like this: `gofmt -s -w config/.*`